### PR TITLE
Update hackathon-march-2021.md

### PR DESCRIPTION
**Pull request by renbot-bio**
Updated with the CoC

### CODE REVIEW

File: markdown/events/2021/hackathon-march-2021.md [hunk 1]
```diff
@@ -44,6 +44,84 @@ If you register in time we will send you some goodies in the post!
...
+
+In your email report, please do your best to include:
+
+- Your contact information.
+- Identifying information (e.g. names, nicknames, pseudonyms) of the participant who has violated the Code of Conduct.
+- The behaviour that was in violation and the circumstances surrounding the incident.
+- The approximate time of the behaviour (if different than the time the report was made)
+- Other people involved in the incident if applicable.
+- If you believe the incident is ongoing, please let us know.
+- If there is a publicly available record (e.g. mailing list record, a screenshot), please include a link.
```
[the remark this answers is not in the document]
Comment (ewels) [reply markdown/events/2021/hackathon-march-2021.md:82]:
```suggestion
- If there is a publicly available record (e.g. mailing list record, a screenshot - please attach if so).
```

**Review by ewels — COMMENTED** [review]
Text looks great! My only comment is it's position in the document. It's quite long and it's right near the top, where I think the most critical overview information should be (overview of what the hackathon is about and how to get involved). Is it ok if we move it below this stuff? The page has a table of contents sidebar so it shouldn't get lost.

**Comment by renbot-bio** [comment]
Yes, good with moving it down on the page. I'm good just as long as it's there and people know what to do in case it's required. 
Thanks for the commits.

**Review by ewels — APPROVED** [review]

**Comment by ewels** [comment]
Live: https://nf-co.re/events/2021/hackathon-march-2021#safety-information-at-the-hackathon

Rendering is a bit funky, looks like an unclosed tag somewhere. Any ideas @mashehu ?

**Comment by mashehu** [comment]
Yes, the table somehow keeps breaking things... 😠

**Comment by ewels** [comment]
I guess it's something to do with this:

https://github.com/nf-core/nf-co.re/blob/e1305a4125150807dc4eba01022dc88c9a8277d2/includes/parse_md.php#L99-L101

The first `replace` won't work as we have added custom classes to this event table. But the second `replace` will work, which means we end up adding an extra `</div>`. I'll submit a PR to fix now 👍🏻

**Comment by mashehu** [comment]
I actually had added some code to catch this (without having found the initial culprit: 
https://github.com/nf-core/nf-co.re/blob/master/public_html/events.php#L188-L191

**Comment by ewels** [comment]
Fix 👉🏻  https://github.com/nf-core/nf-co.re/pull/679

**Comment by ewels** [comment]
> I actually had added some code to catch this (without having found the initial culprit):
> https://github.com/nf-core/nf-co.re/blob/master/public_html/events.php#L188-L191

Correct me if I'm wrong, but does that not do the opposite - adds additional closing `</div>` tags? We already have too many here so that will just make it worse 😅

**Comment by ewels** [comment]
Next one @mashehu - any ideas why the bottom 3 ToC items appear to be a lower hierarchy level?



They are all `h2` headings in the markdown..

**Comment by mashehu** [comment]
No, because it adds the extra '</div>` for the `col` div, only if they divs in the parsed markdown are balanced, which wasn't the case with the table.

**Pull request merged**